# metagenomics_abundance_normalize
a metagenomics abundance normalizer which will take the abundance OTUs file and gives you a normalized ratio for plotting of the species. Normally when you plot OTUs, the OTU graph doesnt make sense as there are lot of OTUs and the clumpsy graph, this approach, takes a species abundance and then divide it by the total number of the OTUs associates with the  species and gives you a number. You can plot as many species as you want and the analysis will look better and meaningful.

 ```
metagenomicsAbundanceNormalizer
                ("/Users/gauravsablok/Desktop/GitHub/shell_plotter/taxonomy_OTUs", \
                                                   "Clostridia", "100")
metagenomicsAbundanceNormalizer
             ("/Users/gauravsablok/Desktop/GitHub/shell_plotter/taxonomy_OTUs", \
                                                  "Porphyromonadaceae", "200")
2536.0
```



Gaurav Sablok \
Frontiers: https://loop.frontiersin.org/people/33293/overview \
ORCID: https://orcid.org/0000-0002-4157-9405 \
WOS: https://www.webofscience.com/wos/author/record/C-5940-2014 \
Github:https://github.com/sablokgaurav \
Linkedin: https://www.linkedin.com/in/sablokgaurav/ \
RubyGems Published: https://rubygems.org/profiles/sablokgaurav \
Python Packages Published : https://pypi.org/user/sablokgaurav/
